index.bs

```diff
@@ -3199,6 +3494,176 @@ The [=remote end steps=] with |command parameters| are:
 1. Return a new map matching the <code>ScriptEvaluateResultSuccess</code>
    production, with the <code>result</code> field set to |result|.
 
+#### The script.callFunction Command ####  {#command-script-callFunction}
+
+The <dfn export for=commands>script.callFunction</dfn> command calls a provided
+function with given arguments in a given realm.
+
+<code>RealmInfo</code> can be either a realm or a browsing context.
+
+Note: <code>this</code> argument is ignore in case of arrow function in
```

Comment:
`The <code>this</code> argument`

Also, it's nto quite ignored; if it doesn't resolve to an object we throw an error, but it doesn't affect the `this` binding inside the function.

Reply:
Fixed note.